src/scaling.jl

```diff
@@ -12,93 +12,66 @@ Represents the no-op scaling which simply returns the `data` it is applied on.
 """
 struct IdentityScaling <: AbstractScaling end
 
-_apply(x, scaling::IdentityScaling; kwargs...) = x
-_apply!(x, scaling::IdentityScaling; kwargs...) = _apply(x, scaling; kwargs...)
-
+@inline _apply(x, ::IdentityScaling; kwargs...) = x
+@inline _apply!(x, ::IdentityScaling; kwargs...) = x
```

Comment:
Is this needed? Can we test this line if yes?

Reply:
you're talking about the `_apply!` method - yeah we probably don't need it